core/src/Kokkos_Serial.hpp

```diff
@@ -249,12 +249,22 @@ struct ZeroMemset<
       : Base(dst, value) {}
 };
 
+template <typename MemorySpace>
+struct MemorySpaceAccess<MemorySpace, Kokkos::Serial::scratch_memory_space> {
+  enum : bool { assignable = false };
+  enum : bool {
+    accessible =
+        MemorySpaceAccess<MemorySpace, Kokkos::Serial::memory_space>::accessible
+  };
+  enum : bool { deepcopy = false };
+};
```

Comment:
Why do you have this? The other host backends (OpenMP and Threads) don't need this.

Reply:
Apparently, we don't need this anymore after #4896.